src/utils/alimentaide/onStructureAdded.ts

```diff
@@ -73,6 +73,28 @@ const ROLE_ID_MAP = {
   userPlus: new Types.ObjectId('651157468e4cb3d8a3f22a7a'),
 };
 
+/**
+ * Adds the context filter of the base app to the structure apps
+ * @param structureApps The structure apps
+ */
+const addContextFilterToStructureApps = async (structureApps) => {
+  // Filter out the base app from the structure apps
+  const appsWithoutBaseApp = structureApps.filter(
+    (a) => !a._id.equals(BASE_APP_ID)
+  );
+  // We fetch the base app
+  const baseApp = await Application.findById(BASE_APP_ID).populate({
+    path: 'pages',
+    model: 'Page',
```

Comment:
Doesn't make sense to populate here if you're not using the pages
https://mongoosejs.com/docs/populate.html